### PR TITLE
feature(#82): add google-analytics

### LINKED ISSUE #81: Set Side/Type Filter via Icons on Strat
- [ ] Click on Side / Type Icon on Strat will set Filter accordingly

### LINKED ISSUE #82: Switch Analytics to Google Analytics
- [ ] Move from Splitbee to Google Analytics

### PR DESCRIPTION
**Pull request by JH1ller**
+ feature(#81): set filter on strat item icon click
+ chore: hide splitbee from menu

closes #82 
closes #81

### CODE REVIEW

File: client/src/components/menus/MainMenu/MainMenu.ts [hunk 1]
```diff
@@ -66,7 +65,7 @@ export default class MainMenu extends Vue {
       }),
       () => {
         // TODO: consider if we should just insert current version here
-        window.open('https://csgo-stratbook.s3.eu-central-1.amazonaws.com/Stratbook+Setup+1.8.4.exe');
+        window.open('https://csgo-stratbook.s3.eu-central-1.amazonaws.com/Stratbook+Setup+1.8.3.exe');
```
Comment (ItsClemi) [inline client/src/components/menus/MainMenu/MainMenu.ts:68]: Is there a reason behind this change?

File: client/src/config.ts [hunk 2]
```diff
@@ -11,3 +11,7 @@ export const STATIC_URL =
 export const S3_URL = 'https://csgo-stratbook.s3.amazonaws.com/';
 
 export const TOKEN_TTL = 1 * 60 * 60 * 1000; // 1h
+
+export const SPLITBEE_ID = 'J3KX6SRRBPBD';
+
+export const GA_ID = 'G-787NDTZVPN';
```
Comment (ItsClemi) [inline client/src/config.ts:17]: Shall we inject this via .env, in the future?

**Review by ItsClemi — APPROVED** [review]

File: client/src/components/menus/MainMenu/MainMenu.ts (hunk 1 above)
Comment (JH1ller) [reply client/src/components/menus/MainMenu/MainMenu.ts:68]: 1.8.4 doesn't exist yet on S3, only have my macbook with me and can't build NSIS on it :P

File: client/src/config.ts (hunk 2 above)
Comment (JH1ller) [reply client/src/config.ts:17]: Yes, most definately. Afaik, on the nest_backend branch the FE already uses .env instead of this TS file.

**Pull request merged**